### PR TITLE
Fixed support to ukraine sticking to screen

### PR DESCRIPTION
**Pull request by h3cksamrat**

### CODE REVIEW

**Comment by chadaustin** [comment]
@h3cksamrat Thanks for the report and the patch. Can you explain the symptom you're seeing? I'm unable to reproduce -- maybe.

**Comment by h3cksamrat** [comment]
First, the banner looked  like this
Support Banner

After Scrolling, the website looks like
afterScrolling

After applying this fix
fixedScrolling

File: website/_sass/_base.scss [hunk 1]
```diff
@@ -60,6 +60,7 @@ h1, h2, h3, h4 {
   transition: opacity 0.2s ease-in-out;
   width: 100%;
   z-index: 9999;
+  top: 60px;
```
Comment (h3cksamrat) [inline website/_sass/_base.scss:63]: Leaves the space for socialBanner to be viewed

**Comment by h3cksamrat** [comment]
@dmitryvinn-fb I have included screenshots

**Pull request closed without merging**